Clinical-Genomics/cg
# Decouple beacon

This PR removes beacon-related commands from cg.

### How to prepare for test
- [x] ssh to hasta (depending on type of change)
`bash /home/proj/stage/servers/resources/hasta.scilifelab.se/update-cg-stage.sh deacouple_beacon`
- [x] use the stage environment with the `us` command

### How to test
beacon-related commands that are on current master branch should not be present any more. Specifically:
- `cg upload --help`

![image](https://user-images.githubusercontent.com/28093618/91858562-a3ad8280-ec69-11ea-99e2-097ea32fc28c.png)


- `cg clean --help`

![image](https://user-images.githubusercontent.com/28093618/91858871-0c94fa80-ec6a-11ea-8706-ded720ad540a.png)


- cg upload beacon --help

![image](https://user-images.githubusercontent.com/28093618/91859055-47972e00-ec6a-11ea-8728-c24b3742e9ae.png)


- cg clean beacon --help

![image](https://user-images.githubusercontent.com/28093618/91859210-71505500-ec6a-11ea-9214-7f376f881ca4.png)


### Expected test outcome
- [x] The commands above should not be found

## Review
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
